class.application-passwords.php

```diff
@@ -86,7 +86,11 @@ public static function show_user_profile( $user ) {
 
 			<?php if ( $new_password ) : ?>
 			<p class="new-application-password">
-				<?php printf( __( 'Your new password for <strong>%s</strong> is <kbd>%s</kbd>.', 'two-factor' ), esc_html( $new_password_name ), self::chunk_password( $new_password ) ); ?>
+				<?php printf(
+					esc_html_x( 'Your new password for %1$s is %2$s.', 'application, password', 'two-factor' ),
+					'<strong>' . esc_html( $new_password_name ) . '</strong>',
+					'<kbd>' . self::chunk_password( $new_password ) . '</kbd>'
```

Comment:
Since we're doing it anyways, should we also pass an `esc_html()` around the result in this row?